src/WebUI/Components/DaemonSetListingComponent.tsx

```diff
@@ -26,97 +27,94 @@ export class DaemonSetListingComponent extends BaseComponent<IDaemonSetComponent
         return (
             <div>{
                 <ListComponent
-                    className={css("list-content", "top-padding", "depth-16" )}
-                    items={this.props.daemonSetList.items || [] }
+                    className={css("list-content", "top-padding", "depth-16")}
+                    items={this.props.daemonSetList.items || []}
                     columns={DaemonSetListingComponent._getColumns()}
-                    onRenderItemColumn={DaemonSetListingComponent._onRenderItemColumn}
                 />
             }</div>
         );
     }
 
-    private static _getColumns(): IColumn[] {
-        let columns: IColumn[] = [];
+    private static _getColumns(): ITableColumn<V1DaemonSet>[] {
+        let columns: ITableColumn<V1DaemonSet>[] = [];
         const headerColumnClassName: string = "secondary-text";
 
         columns.push({
-            key: setNameKey,
+            id: setNameKey,
             name: Resources.DaemonSetText,
-            fieldName: setNameKey,
-            minWidth: 250,
-            maxWidth: 250,
+            width: 250,
             headerClassName: headerColumnClassName,
-            columnActionsMode: ColumnActionsMode.disabled
+            renderCell: (rowIndex: number, columnIndex: number, tableColumn: ITableColumn<V1DaemonSet>, daemonSet: V1DaemonSet) => this._renderSetNameCell(rowIndex, columnIndex, tableColumn, daemonSet),
```

Comment:
rather calling a specific function from here, would it be good if we call a general function and in it if we make the appropriate redenring? just a thought, your version of change is also fine with me.